tracing log along diff crates (#1640)
rust-lang/rustc-dev-guide: src/tracing.md

# compilers.
$ RUSTC_LOG=rustc_codegen_ssa=info rustc +stage1 my-file.rs

> # This will show the all tracing calls under `rustc_codegen_ssa` and
> # `rustc_resolve`.

Reviewer: ```suggestion
# This will show all logs in `rustc_codegen_ssa` and `rustc_resolve`.
```
